liqd_product/apps/users/templates/liqd_product_users/indicator.html

```diff
@@ -23,6 +23,10 @@
                 <a class="dropdown-item" href="{% url 'account' %}">{% trans "Account Settings" %}</a>
             </li>
 
+            <li>
+                <a class="dropdown-item" href="{% url 'profile' request.user.username %}">{% trans 'view your profile' %}</a>
```

Comment:
Maybe we should write the first Letter capitalized, like on the other buttons/links in the user dropdown?